materialrangebar/src/main/java/com/appyvet/materialrangebar/Bar.java

```diff
@@ -175,15 +256,79 @@ public void setTickCount(int tickCount) {
      * @param canvas Canvas to draw on; should be the Canvas passed into {#link
      *               View#onDraw()}
      */
-    public void drawTicks(Canvas canvas) {
+    public void drawTicks(Canvas canvas, float pinRadius, PinView rightThumb) {
+        drawTicks(canvas, pinRadius, rightThumb, null);
+    }
+
+    public void drawTicks(Canvas canvas, float pinRadius, PinView rightThumb, @Nullable PinView leftThumb) {
+        boolean paintLabel = false;
+        if (mLabelPaint != null) {
+            paintLabel = true;
+            final int textSize = (int) TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_SP, 12,
+                    mRes.getDisplayMetrics());
+            mLabelPaint.setTextSize(textSize);
+        }
 
         // Loop through and draw each tick (except final tick).
-        for (int i = 0; i < mNumSegments; i++) {
+        int i = 0;
+        for (;i < mNumSegments; i++) {
```

Comment:
fix this

Reply:
The button label in the sample were cut because of small mrb_rangeBarPaddingBottom
the developers whom will use this need to set the mrb_rangeBarPaddingBottom / layout_height /  mrb_tickLabelSize respectively

Reply:
I added mrb_tickLabelSize & fixed crash if mrb_tickDefaultLabel isn't set (use "")